lib/phoenix/endpoint/cowboy_handler.ex

```diff
@@ -48,7 +48,7 @@ defmodule Phoenix.Endpoint.CowboyHandler do
   end
 
   defp info(scheme, endpoint, config) do
-    "Running #{inspect endpoint} with Cowboy on port #{inspect config[:port]} (#{scheme})"
+    "Running #{inspect endpoint} with Cowboy on #{scheme}://localhost:#{config[:port]}"
```

Comment:
Maybe we could just call `endpoint.url` and it would retrieve the whole thing properly?